src/components/FeatureViews/Accounts/Accounts.js

```diff
@@ -300,13 +299,14 @@ class Accounts extends Component {
                                 indicatorColor="primary"
                                 onChange={this.handleChange}
                                 textColor="primary"
+                                scrollButtons="off"
```

Comment:
This is necessary for mobile view. Figure out how to only enable this on mobile view

Reply:
I thought we were not prioritizing mobile view at this time

Reply:
good point - but we still need to create a ticket for this so we don't forget. Go ahead and create a ticket to address this later.